README.md

```diff
@@ -119,3 +119,10 @@ rewrite the golden files when you run it.
 
 Export the environment variable `TEST_FILTER`, a regex, to limit the end-to-end
 tests (found in `test_files/...`) run tests with a name matching the regex.
+
+### Releasing
+
+On a new branch, run `npm version <major|minor|patch|...> -m 'rel: %s'` (see
+`npm help version` for details) to update the version in `package.json`, commit
+the changes, and create a git tag. Push the branch, get it reviewed and merged,
```

Comment:
"create a git tag like v1.2.3" (or is it just "1.2.3" or "release-1.2.3"?)

Reply:
It'll create whatever it creates, we don't have influence on it and we don't really care either, do we?

Reply:
Ah I read it as
on a new branch
1) run npm version
2) commit the changes
3) create a git tag

but I now see your parse too.